tests/test_app/TestApp/TestCase/Event/InvokableEventTestListener.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace TestApp\TestCase\Event;
+
+use Cake\Event\EventInterface;
+use Cake\Event\EventListenerInterface;
+
+/**
+ * Mock class used to test event dispatching
+ */
+class InvokableEventTestListener implements EventListenerInterface
```

Comment:
why not use an anonymous class?

Reply:
We can. I just am oldschool I guess - with like real test files ;)